packages/react-core/src/demos/Card/Card.md

```diff
@@ -1,6 +1,6 @@
 ---
-id: Card
-section: components
+id: Card view
+section: demos
```

Comment:
Please update the screenshots: `yarn build:docs && yarn serve:docs` then in a separate terminal `yarn screenshots`.

Reply:
I'm getting an error with the `screenshots` command, says it doesn't find the command?

Reply:
Ah sorry, try `yarn workspace @patternfly/react-docs screenshots`. Please add `"screenshots": "yarn workspace @patternfly/react-docs screenshots",` so you just have to run `yarn screenshots` in the future!

Reply:
Thanks that did the trick! Updated.

Reply:
Added the command to the package.json as well